ui/src/main/webapp/app/query/input-manager/seed-builder/seed-builder-component.js

```diff
@@ -80,7 +80,7 @@ function SeedBuilderController(schema, csv, types, error, events, common, $route
      * Creates the placeholder for the seed input
      */
     vm.getPlaceHolder = function() {
-        return vm.usePrevious ? "Input is provided by the output of the previous operation" : "Enter your seeds, each seed on a new line\n" + vm.getCsvHeader()
+        return vm.usePrevious ? "Input is provided by the output of the previous operation" : "Enter your seeds, each seed on a new line: \n" + vm.getCsvHeader()
```

Comment:
This would look strange in the event that there is an empty csv header. For example if using string seeds.

Reply:
Good point - I've removed the :
The space is useful in case the browser doesn't display the \n as a new line.